variables.tf

```diff
@@ -244,7 +253,7 @@ variable "routes" {
     }), {})
 
     # Integration
-    integration = object({
+    integration = map(object({
```

Comment:
we want to have integration attached to multiple routes, we don't need this as map

Reply:
i think we need to remove integration map from here and create integrations variable, which is map of integrations then we pass needed integration to the route